index.html

```diff
@@ -0,0 +1,168 @@
+        <h2>ABREHOT Library</h2>
+        <div class="p-2 border-1 bg-gray">
+          <p class="p-1">
+            "Embark on a journey of discovery and endless learning as you
+            immerse yourself in the boundless treasures of knowledge and
+            imagination available at your fingertips in our library."
+          </p>
+        </div>
+        <p class="tbm-header bg-gray">25<sup>th</sup> May 2023</p>
+        <p class="nyerere">@Arat Killo, Addis Ababa</p>
+      </section>
+      <!-- Home page Main Part -->
+      <section class="main-part font">
+        <h2>OUR SERVICES</h2>
+        <div class="dash">_____</div>
+        <div class="main-div">
+          <div>
+            <img src="images/book_borrowing.png" alt="book Logo" />
+            <h3>Borrow</h3>
+            <p>Books are a uniquely portable magic.Enjoy by being Member.</p>
+          </div>
+          <div>
+            <img src="images/audio-book.png" alt="Audiobook Logo" />
+            <h3>Audio Book</h3>
+            <p>
+              An audiobook is a great way to escape into a story while you're
+              doing other things.
+            </p>
+          </div>
+          <div>
+            <img src="images/place.jpg" alt="place" />
+            <h3>Corner</h3>
+            <p>
+              The best place to study, quiet, well-lit, comfortable, and free
+              from distractions.
+            </p>
+          </div>
+          <div>
+            <img class="bg-red" src="images/coffee.jpg" alt="Coffee" />
+            <h3>Coffee</h3>
+            <p>
+              When you drink coffee ideas march in like the army.Enjoy our
+              Coffee.
+            </p>
+          </div>
+          <div>
+            <img src="images/internet.jpg" alt="Internet" />
+            <h3>Internet</h3>
+            <p>
+              Enjoy our High Bandwidth Internet.we support wifi and Ethernet.
+            </p>
+          </div>
+        </div>
+        <div class="last-main-div">
+          <h3 class="bg-red h3-one">LIBRARIES FUEL IMAGINATION</h3>
+        </div>
+        <h3>SEE OUR LIBRARY RESOURCE</h3>
+      </section>
+      <!-- Featured Partcipants -->
+      <section class="investors font">
+        <h2>Featured Advisors</h2>
+        <div class="dash">_____</div>
+        <div class="partcipants"></div>
+        <button class="none">
+          MORE <i class="fa-solid fa-greater-than fa-rotate-90"></i>
+        </button>
+      </section>
+      <section class="none-mobile last-section">
+        <h2>Partner</h2>
+        <div class="dash">____</div>
+        <ul class="partner-div">
+          <li id="barclays">SCIENCE MUSEUM</li>
+          <li id="toyota">ETHIO TELE</li>
+          <li id="star">STAR</li>
+          <li id="oil">OIL</li>
+          <li id="google">GOOGLE</li>
+        </ul>
+      </section>
+    </main>
+    <footer class="home-footer">
+      <h3>ABREHOT</h3>
+      <p>Home of Free Thinkers.</p>
+    </footer>
```

Comment:
- [ ] [OPTIONAL] Please make sure that your footer design is similar to the original design. There are a few differences. Please note that it is very important to stick to the original design as much as possible :smile: 

#### The original design
![Screenshot from 2022-10-28 18-41-53](https://user-images.githubusercontent.com/6615394/198709533-f9ce48e0-df96-4c8a-b29b-ffea117dcfcf.jpg)